pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -360,6 +361,15 @@ func changefeedPlanHook(
 			}
 		}
 
+		if metrics, ok := p.ExecCfg().JobRegistry.MetricsStruct().Changefeed.(*Metrics); ok {
+			scope, _ := opts.GetMetricScope()
+			sli, err := metrics.getSLIMetrics(scope)
+			if err != nil {
+				return err
+			}
+			sli.CreatePTSHistNanos.RecordValue(timeutil.Since(ptsStart).Nanoseconds())
```

Comment:
Do we want to differentiate between "creating" and "managing"? There are times when we create a new PTS in the `manageProtectedTimestamps` function that you also modified.